swap_meet/vendor.py

```diff
@@ -1,2 +1,120 @@
+
 class Vendor:
-    pass
\ No newline at end of file
+
+    # Creator: Each Vendor object has an attribute inventory
+    # The inventory should be empty if the caller does not
+    # provide any argument.
+    def __init__(self, inventory=None):
+        # If the caller did not provide an inventory,
+        # then create a new empty list and save in the new
+        # Vendor's object inventory
+        if not inventory:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+    # method: add item to inventory
+    # the argument item is a reference to an object of class Item
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    # method: remove item from inventory
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
```

Comment:
Let's switch these. We want the "main" focus of the code on the bottom of the guard clause:

```suggestion
    def remove(self, item):
        if item not in self.inventory:
            return False
        self.inventory.remove(item)
        return item
```